src/main/resources/application.yml

```diff
@@ -44,6 +44,7 @@ madie:
   qdm-service:
     base-url: ${QDM_SERVICE_URL:http://localhost:8086/api}
     create-package-urn: /qdm/measures/package
+    create-qrda-urn: /qdm/measures/package/qrda
```

Comment:
I think this should be its own block e.g.

```
qrda-service:
   base-url: ${QRDA_SERVICE_URL:http://localhost:8087/api}
   create-qrda-urn: /qrda
 
```
or 
`qrda-service: ${QRDA_SERVICE_URL:http://localhost:8087/api/qrda}` 

please check qrda service for the path

Reply:
gotcha, thanks!